app/src/main/res/layout/item_message.xml

```diff
@@ -16,14 +16,38 @@
         android:layout_height="40dp"
         android:layout_marginTop="7dp"
         app:layout_constraintLeft_toLeftOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+        app:layout_constraintTop_toBottomOf="@id/new_messages_notif" />
+
+    <LinearLayout
+        android:id="@+id/new_messages_notif"
+        tools:visibility="visible"
+        android:visibility="gone"
+        android:orientation="horizontal"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent">
+        <View
+            android:layout_gravity="center"
+            android:layout_height="1dp"
+            android:layout_width="0dp"
+            android:layout_weight="1"
+            android:layout_marginRight="4dp"
+            android:background="@color/red"/>
+        <TextView
+            android:layout_width="wrap_content"
+            android:text="unread messages"
```

Comment:
I believe we should use i18n strings here?

Reply:
I'll add it to the next commit, together with any other review issues.